.vscode/launch.json

```diff
@@ -40,6 +40,33 @@
       "sourceMaps": true,
       "outFiles": ["${workspaceRoot}/packages/*/out/src/**/*.js"]
     },
+    {
+      "name": "Launch Extensions (use current LWC)",
+      "type": "extensionHost",
+      "request": "launch",
+      "runtimeExecutable": "${execPath}",
+      "args": ["--extensionDevelopmentPath=${workspaceRoot}/packages"],
+      "stopOnEntry": false,
+      "sourceMaps": true,
+      "outFiles": ["${workspaceRoot}/packages/*/out/src/**/*.js"],
+      "preLaunchTask": "Compile",
+      "env": {
+        "FORCE_LOAD_CURRENT_LWC": "true"
```

Comment:
![screen shot 2018-03-08 at 10 17 49 am](https://user-images.githubusercontent.com/54414/37168519-0561db10-22ba-11e8-95fb-b7c07b0d9f0a.png)

By default, we will prefer to launch the lwc-next. But you can use this other configuration for testing locally.